.gitignore

```diff
@@ -1,6 +1,7 @@
 /.php-cs-fixer.cache
 /.phpunit.cache/
 /composer.lock
-/files/decode/input.php
-/files/decode/output.php
 /vendor/
+
+/files/decoder/input.smile
+/files/decoder/output.json
```

Comment:
Theses 2 files are in the .gitignore, but are ALSO commited. This is strange. what's the use case of theses files?

Reply:
These files are here because they are used by the commands. 
However their content shouldn't be commited, so I commited them empty then added them to the gitignore.

They are probably not required and could be removed, but it would mean that users would need to create an input file and make the command read it. This is not necessarily a bad thing, especially when considering that IDEs may overwrite the binary file on saving.